source/uwp/AdaptiveCardTestApp/Expected/SourceCards/v1.0.Elements.AdaptiveCard.50d0fa6.json

```diff
@@ -0,0 +1,132 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "Container",
+			"items": [
+				{
+					"type": "TextBlock",
+					"text": "Publish Adaptive Card schema",
+					"weight": "bolder",
+					"size": "medium"
```

Comment:
>	 [](start = 0, length = 1)

spaces instead of tabs?

Reply:
I'm not sure how these files are auto generated. Let me see if I can find that code in the testApp.

---
In reply to: [150308541](https://github.com/Microsoft/AdaptiveCards/pull/921#discussion_r150308541) [](ancestors = 150308541)

Reply:
The sample files are using tabs instead of spaces. Changing those.

---
In reply to: [150308968](https://github.com/Microsoft/AdaptiveCards/pull/921#discussion_r150308968) [](ancestors = 150308968,150308541)